package.json

```diff
@@ -47,7 +47,9 @@
     "webpack-cli": "^5.0.1"
   },
   "workspaces": [
-    "js/packages/binderhub-client"
+    "js/packages/binderhub-client",
+    "js/packages/binderhub-react-components",
+    "js/packages/binderhub-spec"
```

Comment:
can we put this in binderhub-client too? react-component already imports that too right?

Reply:
I'm not 100% sure if I understood this correct. Please check fbe7466f41ea32e56c287a4a63c129132b095f6c.

My understanding is that the dependence tree is

```mermaid
flowchart LR
    binderhub-client --> binderhub-react-components
    binderhub-spec --> binderhub-react-components
    binderhub-react-components --> App
    binderhub-spec --> App
```